client/webserver/site/.stylelintrc

```diff
@@ -12,6 +12,8 @@
         "scss/double-slash-comment-empty-line-before": null,
         "scss/at-import-no-partial-leading-underscore": null,
         "scss/at-import-partial-extension": null,
-        "scss/at-extend-no-missing-placeholder": null
+        "scss/at-extend-no-missing-placeholder": null,
+        "declaration-block-no-redundant-longhand-properties": null,
+        "value-no-vendor-prefix": null
```

Comment:
Why these changes?

Reply:
To allow for changes fixing  https://github.com/decred/dcrdex/pull/2333#issuecomment-1546698329